dotCMS/src/main/java/com/dotcms/storage/StorageKey.java

```diff
@@ -0,0 +1,115 @@
+package com.dotcms.storage;
+
+import java.io.Serializable;
+
+/**
+ * Encapsulates a storage key, it has the key (for instance the path)
+ * the group (for instance the bucket, folder, space, etc)
+ * Storage {@link StorageType}
+ * @author jsanca
+ */
+public class StorageKey implements Serializable {
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6220272)